src/legacy/core_plugins/tile_map/public/__tests__/coordinate_maps_visualization.js

```diff
@@ -31,11 +31,12 @@ import EMS_TILES from '../../../../ui/public/vis/__tests__/map/ems_mocks/sample_
 import EMS_STYLE_ROAD_MAP_BRIGHT from '../../../../ui/public/vis/__tests__/map/ems_mocks/sample_style_bright';
 import EMS_STYLE_ROAD_MAP_DESATURATED from '../../../../ui/public/vis/__tests__/map/ems_mocks/sample_style_desaturated';
 import EMS_STYLE_DARK_MAP from '../../../../ui/public/vis/__tests__/map/ems_mocks/sample_style_dark';
-import { setup as visualizationsSetup } from '../../../visualizations/public/np_ready/public/legacy';
 
 import { createTileMapVisualization } from '../tile_map_visualization';
 import { createTileMapTypeDefinition } from '../tile_map_type';
-import { ExprVis } from '../../../visualizations/public/np_ready/public/expressions/vis';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { ExprVis } from '../../../../../plugins/visualizations/public/expressions/vis';
```

Comment:
I removed `ExprVis` class from the `public/index.ts` because it is not technically static (requires the vis types registry), and instead I'm only exporting `ExprVis` as a type.

However there are two map tests which are currently using the class `new ExprVis()`, hence ignoring `no-restricted-paths` here.